Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Compressors/Compressor.cpp

```diff
@@ -35,6 +33,14 @@ namespace ImageProcessingAtom
                 return ICompressorPtr(new CTSquisher());
             }
         }
+        
+        if (ASTCCompressor::IsCompressedPixelFormatSupported(fmt))
+        {
+            if (isCompressing || (!isCompressing && ASTCCompressor::DoesSupportDecompress(fmt)))
```

Comment:
Since DoesSupportDecompress is always true do you even need to check for isCompressing?

Reply:
What is the implementation of ASTCCompressor::DoesSupportDecompress(fmt) changed? 
I know it's unlikely. But we shouldn't use what we see inside implementation to change the condition here.